scenario_configs/poisoning_baselines/cifar10/dlbd/watermark/cifar10_dlbd_watermark_activation_defense.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
should the `poisoning_baselines` dir go in the `eval5` dir? At its current dir level, not sure it quite fits to sit alongside `eval5` and `eval1-4`

Reply:
I can reason either way.  These could be useful in future evals or outside the gard evaluation program.  On the other hand, they're created especially for convenience in evaluating this specific round of threat models.   I'm happy to do whatever you think makes more sense

Reply:
If there are particular ones that would be good to link from the base directory, though, let's add a link or 2.